lib/crypto_env_var.rb

```diff
@@ -19,5 +37,13 @@ def decrypt(string, public_key_string)
       json = cipher.decrypt(encrypted_data)
       Utils.deserialize(json)
     end
+
+
```

Comment:
Nit: extra newline.

Reply:
I know that this is debatable, but a while ago [we agreed](https://deliveroo.engineering/guidelines/ruby/#empty-lines) that 2 empty lines outside of method bodies are ok, because it's visually different from the single empty lines used to break a method. It's like _tabs Vs spaces_, agreed, but I think that that's the convention we've been trying to stick to.

In this specific case I just find it visually unpleasant to have 2 lines between methods and just one line between methods and `private`. 😄

Reply:
(The guidelines are in a public repo and anyone can send a PR to amend them or propose something new)